rust/kcl-lib/src/main.rs

```diff
@@ -44,4 +44,5 @@ async fn main() {
     .unwrap();
     let mut exec_state = ExecState::new(&ctx);
     ctx.run(&program, &mut exec_state).await.map_err(|e| e.error).unwrap();
+    println!("{:#?}", exec_state.errors());
```

Comment:
should probably be `eprintln` and should be wrapped in `if !exec_state.errors().is_empty()`